include/seqan3/utility/char_operations/predicate_detail.hpp

```diff
@@ -15,10 +15,11 @@
 
 #include <array>
 #include <cctype>
-#include <seqan3/std/concepts>
 #include <stdexcept>
 #include <string>
 
+#include <seqan3/std/concepts>
+
```

Comment:
Why did you move this down here? I understood that we sort it as if it was `<concepts>`...

Reply:
whoops. good catch. I don't know what happend